app/src/main/res/values/strings.xml

```diff
@@ -1780,4 +1780,13 @@
     <string name="places_card_distance_unknown">Unknown distance</string>
     <string name="places_empty_message_snackbar">There are no nearby articles. Try zooming out and moving the map.</string>
     <!-- /Places -->
+
+    <!-- Recommended Content -->
+    <string name="recommended_content_section_you_might_like">You might like</string>
+    <string name="recommended_content_survey_dialog_title">Help improve recommended content.</string>
+    <string name="recommended_content_survey_dialog_message">Are you satisfied with this recommended search result?</string>
+    <string name="recommended_content_survey_dialog_input_hint">Additional Feedback</string>
```

Comment:
Minor grammar: the second word `Feedback` should not be capitalized.

Reply:
Updated.